tests/circleci.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+i=0
+tests=()
+exit_code=0
+IFS=',' read -ra flavors <<< "$CIRCLECI_FLAVORS"
+for flavor in $flavors
```

Comment:
Why are you using a `bash` script when you have the power and the ease of `ruby` and `rake` ?
For instance https://github.com/degemer/dd-agent/blob/7b529d4fb89277226f21ac3edddcdec103f433a9/ci/resources/circleci.rb#L30-L44 (I was taking the flavors from the `.travis.yml` file, not wanted here, but I guess you could do the same with a `Rake::Task['ci:run_circle'].invoke(flavor)`)